tests/benchmarks/test_tpch.py

```diff
@@ -0,0 +1,315 @@
+import os
+from datetime import datetime
+
+import dask.dataframe as dd
+import pytest
+
+DATASETS = {
+    "scale 100": "s3://coiled-runtime-ci/tpch_scale_100/",
+}
+
+enabled_dataset = os.getenv("TPCH_SCALE")
+if enabled_dataset is not None:
+    if enabled_dataset not in DATASETS:
+        raise ValueError("Unknown tpch dataset: ", enabled_dataset)
+else:
+    enabled_dataset = "scale 100"
+
+
+def read_data(filename):
+    path = DATASETS[enabled_dataset] + filename + "/"
+    return dd.read_parquet(path, engine="pyarrow")
+
+
+@pytest.mark.client("tpch")
+def test_query_1(client):
+    VAR1 = datetime(1998, 9, 2)
+    lineitem_ds = read_data("lineitem")
+
+    lineitem_filtered = lineitem_ds[lineitem_ds.l_shipdate <= VAR1]
```

Comment:
I think this is a little bit biased towards dask-expr. Ordinarily, we'd point users to using filters in the read_parquet call.
That's fine, I think, just wanting to point it out

Reply:
> Ordinarily, we'd point users to using filters in the read_parquet call.

I'm pretty comfortable with what's here.  I rarely see users use the read_parquet kwargs in practice, even though they should.  

More generally, I think that we should write things the way we think a naive users would write them, rather than the way they should write them.

Reply:
Actually, I don't think they should. I've done some measurements with filters in read_parquet and they are terribly slow if you don't have metadata available for the columns you are filtering. 

I've explicitly avoided pushing the filters into read_parquet in my benchmarks because that was at least 3 times as slow as doing this with dask/pandas

Reply:
Well, that's sad and we should investigate. If this is indeed slower, `read_parquet` is trying to be smart where it shouldn't be or smth is just implemented poorly.

Reply:
The slowdown is in arrow itself. We aren't doing anything fancy except passing the filters in.